### PR TITLE
Updated lib/sso_openid/urls.yml for auctions

### PR DESCRIPTION
**Pull request by guillec**
Added auctions key and values to the yml.

### CODE REVIEW

File: lib/sso_openid/urls.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 defaults:
+  auctions:
+    host: "auctions.networkforgood-beta.com"
+    fqdn: "https://auctions.networkforgood-beta.com"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:4]: I think this should be fqdn: "https://api.auctions.networkforgood-beta.com". In order for a request to get to the new auctions applications, it needs a subdomain, then "auctions", then the nfg domain. 

This endpoint doesn't exist yet on auctions, but it is ok to include it here for now

File: lib/sso_openid/urls.yml [hunk 2]
```diff
@@ -21,13 +24,19 @@ defaults:
...
   dms:
     host: "dm.n4g.be:3001"
     fqdn: "http://dev.dm.n4g.be:3001"
   evo:
     host: "n4g.be:3000"
     fqdn: "http://basic.n4g.be:3000"
 development:
+  auctions:
+    host: "auctions.test"
+    fqdn: "http://dev.auctions.test"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:39]: is it an entity with a subdomain of "dev" that the seeds create? Feels like it should be "basic" as that is what is created in the FP seeds, and as we discussed yesterday, the subdomains of matching entities in FP and Auctions have to match

File: lib/sso_openid/urls.yml [hunk 3]
```diff
@@ -38,6 +47,9 @@ development:
     host: "fp.test"
     fqdn: "http://basic.fp.test"
 demo:
+  auctions:
+    host: "auctions.networkforgood-demo.com"
+    fqdn: "https://sso.auctions.networkforgood-demo.com"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:52]: let's use 'api' instead of 'sso'

File: lib/sso_openid/urls.yml [hunk 4]
```diff
@@ -60,6 +72,9 @@ demo:
     dashboard: "https://account-sandbox.networkforgood.org/Npo/Dashboard.aspx"
     disbursements: "https://account-sandbox.networkforgood.org/Npo/AccountManagement/OrganizationInfo.aspx"
 staging:
+  auctions:
+    host: "auctions.networkforgood-demo.com"
+    fqdn: "https://sso.auctions.networkforgood-demo.com"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:77]: api instead of sso

File: lib/sso_openid/urls.yml [hunk 5]
```diff
@@ -82,6 +97,9 @@ staging:
     dashboard: "https://account-sandbox.networkforgood.org/Npo/Dashboard.aspx"
     disbursements: "https://account-sandbox.networkforgood.org/Npo/AccountManagement/OrganizationInfo.aspx"
 qa:
+  auctions:
+    host: "auctions.networkforgood-qa.com"
+    fqdn: "https://sso.auctions.networkforgood-qa.com"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:102]: api instead of sso

File: lib/sso_openid/urls.yml [hunk 6]
```diff
@@ -104,6 +122,9 @@ qa:
     dashboard: "https://account-sandbox.networkforgood.org/Npo/Dashboard.aspx"
     disbursements: "https://account-sandbox.networkforgood.org/Npo/AccountManagement/OrganizationInfo.aspx"
 production:
+  auctions:
+    host: "auctions.networkforgood.com"
+    fqdn: "https://sso.auctions.networkforgood.com"
```
Comment (hoenth) [inline lib/sso_openid/urls.yml:127]: api instead of sso

**Review by hoenth — CHANGES_REQUESTED** [review]
A few quick changes

File: lib/sso_openid/urls.yml (hunk 2 above)
Comment (guillec) [reply lib/sso_openid/urls.yml:39]: I see a `dev`, `basic`,`admin` Entities in the seed file.

Comment (guillec) [reply lib/sso_openid/urls.yml:39]: Based on our conversation dev is ok.

**Comment by guillec** [comment]
Pair reviewed with @hoenth

**Pull request merged**